lang/python/python3-flent/Makefile

```diff
@@ -0,0 +1,44 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python3-flent
+PKG_VERSION:=2.0.1
+PKG_RELEASE:=$(AUTORELEASE)
+
+PYPI_NAME:=flent
+PKG_HASH:=300a09938dc2b4a0463c9144626f25e0bd736fd47806a9444719fa024d671796
+
+PKG_MAINTAINER:=Nick Hainke <vincent@systemli.org>
+PKG_LICENSE:=GPL-3.0-only
+PKG_LICENSE_FILES:=LICENSE
+
+include ../pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-flent
+  SECTION:=lang-python
+  CATEGORY:=Languages
+  SUBMENU:=Python
+  TITLE:=The FLExible Network Tester
+  URL:=https://github.com/tohojo/flent
+  DEPENDS:= \
+    +python3-light \
+    +python3-email \
+    +python3-urllib \
+    +python3-uuid \
+    +python3-logging \
+    +python3-xml \
+    +python3-decimal \
+    +python3-distutils \
+    +netperf \
+    +fping
```

Comment:
There's an existing flent-tools package with the C helper utils. Maybe consolidating that into this package would make sense (and adding a dependency as well)?

Reply:
I would not add it as dependency directly, but maybe as new variant? `flent-with-addons` or something like this?